### PR TITLE
Update react

### PR DESCRIPTION
**Pull request by bilalesi**
1. upgrade react to 18.3 (preparation to 19.0 migration).
2. Fix code editor duplication and code mis-formatting  in studio dashboard (creation).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added necessary unit and integration tests.
- [ ] I have added screenshots (if applicable), in the comment section.

### CODE REVIEW

File: src/shared/components/EditTableForm.tsx [hunk 1]
```diff
@@ -94,6 +95,8 @@ const EditTableForm: React.FC<{
   busy,
   formName = 'Edit',
 }) => {
+  const editorRef = React.useRef<codemirror.Editor>();
+  const wrapperRef = React.useRef(null);
   // state
```
Comment (danburonline) [inline src/shared/components/EditTableForm.tsx:100]: Can this be removed?

File: src/subapps/admin/components/ViewForm/ElasticSearchQueryInput.tsx [hunk 2]
```diff
@@ -4,20 +4,23 @@ import {
   CheckCircleOutlined,
   ExclamationCircleOutlined,
 } from '@ant-design/icons';
-import './ElasticSearchQueryInput.scss';
-
+import codemirror from 'codemirror';
 import { UnControlled as CodeMirror } from 'react-codemirror2';
-import 'codemirror/addon/display/placeholder';
+// import 'codemirror/addon/display/placeholder';
```
Comment (danburonline) [inline src/subapps/admin/components/ViewForm/ElasticSearchQueryInput.tsx:9]: Can this also be removed?

**Review by danburonline — APPROVED** [review]

File: src/shared/components/EditTableForm.tsx (hunk 1 above)
Comment (bilalesi) [reply src/shared/components/EditTableForm.tsx:100]: can not be removed, they are the important thing here 😉 
look at `editorDidMount` and `editorWillUnmount` in the `<CodeMirror/>` component

File: src/subapps/admin/components/ViewForm/ElasticSearchQueryInput.tsx (hunk 2 above)
Comment (bilalesi) [reply src/subapps/admin/components/ViewForm/ElasticSearchQueryInput.tsx:9]: should be back

**Pull request merged**